plugins/animation/test.js

```diff
@@ -0,0 +1,66 @@
+const cssMatcher = require('jest-matcher-css')
+const plugin = require('./index')
+const { generateUtilities } = require('../../testing/generators')
+
+expect.extend({
+  toMatchCss: cssMatcher,
+})
+
+test('it generates the animation classes', () => {
+  const config = {
+    theme: {
+      animation: {
+        spin: {
+          animation: '1s infinite linear',
+          keyframes: {
+            from: { transform: 'rotate(0deg)' },
+            to: { transform: 'rotate(360deg)' },
+          },
+        },
+        fadeOut: {
+          animation: '0.5s forwards',
+          keyframes: {
+            '0%': { opacity: 1 },
+            '100%': { opacity: 0 },
+          },
+        },
+      },
+    },
+  }
+
+  const output = `
+    .animation-spin {
+      animation: spin 1s infinite linear;
+    }
```

Comment:
Just curious do spaces/indentation matter?

Reply:
It does not, neither do semicolons, which is why I missed them in your other comment.
It uses this: https://github.com/benface/jest-matcher-css/blob/master/index.js
Which, funnily enough, was written by the creator of Tailwind